test/utils/decentium.ts

```diff
@@ -0,0 +1,382 @@
+// generated by @greymass/abi2core
```

Comment:
Did you mean to commit this? 😄

Reply:
Yeah the unit tests are using it to test with `Struct` instances from abi2core.